lustre/ws.lua

```diff
@@ -0,0 +1,344 @@
+local cosock = require "cosock"
+local socket = require "cosock.socket"
+local Request = require "luncheon.request"
+local Response = require "luncheon.response"
+local send_utils = require "luncheon.utils"
+local Handshake = require "lustre.handshake"
+local Key = require "lustre.handshake.key"
+local Config = require "lustre.config"
+local Frame = require "lustre.frame"
+local FrameHeader = require "lustre.frame.frame_header"
+local OpCode = require "lustre.frame.opcode"
+local CloseCode = require"lustre.frame.close".CloseCode
+local Message = require "lustre.message"
+local log = require "log"
+
+local utils = require "spec.utils"
```

Comment:
Yeah, I would say this is a bit of an issue, I'd prefer moving it to a utils file in the main part of the project. I'd expect installing this via `luarocks` would fail since we don't have any of the `spec` files included in the output